corehq/apps/userreports/pillow_utils.py

```diff
@@ -78,6 +78,8 @@ def migrate_tables_with_logging(engine, diffs, table_names, adapters_by_table):
     migration_diffs = [diff for diff in diffs if diff.table_name in table_names]
     for table in table_names:
         adapter = adapters_by_table[table]
+        if not adapter.config.is_static:
+            adapter.config.save_build_not_required()
```

Comment:
:+1: 

Can this be done after the actual migration, in the for loop below?
And I believe this won't be needed anymore once the follow up PR is done, which will not set the flag only in case the data source does not need a rebuild.